acsoo/templates/project/+project.name+/pyproject.toml.bob

```diff
@@ -0,0 +1,48 @@
+# Use the hatchling build backend, with the hatch-odoo plugin.
+[build-system]
+requires = ["hatchling", "hatch-odoo"]
+build-backend = "hatchling.build"
+
+# The standard project metadata section
+# https://packaging.python.org/en/latest/specifications/declaring-project-metadata/
+[project]
+name = "odoo-addons-{{{ project.name }}}"
+description = "{{{ project.name|capitalize }}} Odoo Addons"
+version = "{{{ odoo.series }}}.1.0.0"
+readme = "README.rst"
+requires-python = "=={{{ python_version }}}.*"
+# Dependencies are dynamic because they will be generated from Odoo addons manifests.
+dynamic = ["dependencies"]
+
+[project.optional-dependencies]
+test = [
+  "coverage",
+  "pygments",
+  "pytest",
+  "pytest-cov",
+  "pytest-odoo",
+]
+dev = [
+  "watchdog"
+]
+
+[project.scripts]
+
+# Enable the hatch-odoo metadata hook to generate dependencies from addons manifests.
+[tool.hatch.metadata.hooks.odoo-addons-dependencies]
+
+[tool.hatch-odoo]
+# If our addons have non standard version numbers, let's help hatch-odoo discover the Odoo version.
+#odoo_version_override = "16.0"
+# Add dependencies that are not declared in Odoo addons manifests.
+dependencies = [
+  "click-odoo-contrib",
+  # Odoo enterprise addons
+  "odoo-addons-enterprise",
+  # Optional Odoo dependencies not in Odoo's setup.py
+  "freezegun",
+  "num2words",
+  "pdfminer",
+  "xlrd",
+]
+addons_dirs = ["odoo/addons"]
```

Comment:
```
[tool.hatch.build]
packages = ["odoo"]
```

not needed ?

Reply:
@adrienpeiffer `tool.hatch-odoo.addons_dir` is needed because this is where the hatch-odoo metadata plugin looks for addons to generate the dependencies metadata. `tool.hatch.build.packages` is necessary for hatch to include non-python files (see https://github.com/pypa/hatch/issues/478 for the explanation).

Reply:
So we have to add it in order to have XML files in wheels right ?

Reply:
Ah yes, I thought you wanted to remove it :)

Reply:
Indeed. It was not very clear :sweat_smile:

Reply:
done